docs/docs/references/configuration/config-file.md

```diff
@@ -179,346 +79,558 @@ cache:
+```
+## Registry options
 
 ```yaml
-kubernetes:
-  # Same as '--context'
-  # Default is empty
-  context:
+registry:
+  # Same as '--password'
+  # Default is []
+  password: empty
 
-  # Same as '--namespace'
+  # Same as '--registry-token'
   # Default is empty
-  namespace:
+  token:
 
-  # Same as '--kubeconfig'
-  # Default is empty
-  kubeconfig: ~/.kube/config2
+  # Same as '--username'
+  # Default is []
+  username: empty
 
-  # Same as '--components'
-  # Default is 'workload,infra'
-  components: 
-    - workload
-    - infra
+```
+## Rego options
 
-  # Same as '--k8s-version'
-  # Default is empty
-  k8s-version: 1.21.0
+```yaml
+rego:
+  # Same as '--config-check'
+  # Default is []
+  check: empty
 
-  # Same as '--tolerations'
-  # Default is empty
-  tolerations:
-    - key1=value1:NoExecute
-    - key2=value2:NoSchedule
+  # Same as '--config-data'
+  # Default is []
+  data: empty
 
-  # Same as '--all-namespaces'
+  # Same as '--include-deprecated-checks'
   # Default is false
-  all-namespaces: false
+  include-deprecated-checks: false
 
-  node-collector:
-    # Same as '--node-collector-namespace'
-    # Default is 'trivy-temp'
-    namespace: ~/.kube/config2
+  # Same as '--check-namespaces'
+  # Default is []
+  namespaces: empty
 
-    # Same as '--node-collector-imageref'
-    # Default is 'ghcr.io/aquasecurity/node-collector:0.0.9'
-    imageref: ghcr.io/aquasecurity/node-collector:0.0.9
+  # Same as '--skip-check-update'
+  # Default is false
+  skip-check-update: false
 
-  exclude:
-    # Same as '--exclude-owned'
-    # Default is false
-    owned: true
+  # Same as '--trace'
+  # Default is false
+  trace: false
 
-    # Same as '--exclude-nodes'
-    # Default is empty
-    nodes:
-      - kubernetes.io/arch:arm64
-      - team:dev
+```
+## Report options
 
-  # Same as '--qps'
-  # Default is 5.0
-  qps: 5.0
+```yaml
+# Same as '--dependency-tree'
+# Default is false
+dependency-tree: false
 
-  # Same as '--burst'
-  # Default is 10
-  burst: 10
-```
+# Same as '--exit-code'
+# Default is 0
+exit-code: 0
 
-## Repository Options
-Available with git repository scanning (`trivy repo`)
+# Same as '--exit-on-eol'
+# Default is 0
+exit-on-eol: 0
+
+# Same as '--format'
+# Default is table
+format: table
+
+# Same as '--ignore-policy'
+# Default is empty
+ignore-policy:
+
+# Same as '--ignorefile'
+# Default is .trivyignore
+ignorefile: .trivyignore
+
+# Same as '--list-all-pkgs'
+# Default is false
+list-all-pkgs: false
+
+# Same as '--output'
+# Default is empty
+output:
+
+# Same as '--output-plugin-arg'
+# Default is empty
+output-plugin-arg:
+
+# Same as '--report'
+# Default is all
+report: all
+
+scan:
+  # Same as '--compliance'
+  # Default is empty
+  compliance:
+
+  # Same as '--show-suppressed'
+  # Default is false
+  show-suppressed: false
+
+# Same as '--severity'
+# Default is UNKNOWN, LOW, MEDIUM, HIGH, CRITICAL
+severity:
+ - UNKNOWN
+ - LOW
+ - MEDIUM
+ - HIGH
+ - CRITICAL
```

Comment:
We duplicate default values ​​in comment and values.

Maybe we can set a limit on values ​​(like 2 values) to avoid such a big page
e.g.:
```suggestion
# Default is UNKNOWN, LOW, MEDIUM, HIGH, CRITICAL
severity:
 - UNKNOWN
 - LOW
```